File: app/src/main/java/dev/ragnarok/fenrir/db/column/FavePostsColumns.java
package dev.ragnarok.fenrir.db.column;

import android.provider.BaseColumns;

public final class FavePostsColumns implements BaseColumns {

    public static final String TABLENAME = "fave_posts";
    public static final String POST = "post";
    public static final String FULL_ID = TABLENAME + "." + _ID;
    public static final String FULL_POST = TABLENAME + "." + POST;

    private FavePostsColumns() {
    }

}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/db/column/FavePostsColumns.kt
package dev.ragnarok.fenrir.db.column

import android.provider.BaseColumns

object FavePostsColumns : BaseColumns {
    const val TABLENAME = "fave_posts"
    const val POST = "post"
    const val FULL_ID = TABLENAME + "." + BaseColumns._ID
    const val FULL_POST = "$TABLENAME.$POST"
}